pandora-client-web/src/editor/parsing.ts

```diff
@@ -24,19 +24,33 @@ function ParseFloat(input: string): number {
 }
 
 export function SerializeAtomicCondition(condition: Immutable<AtomicCondition>): string {
-	if ('bone' in condition && condition.bone != null)
+	if ('bone' in condition) {
+		Assert(condition.bone != null);
 		return `${condition.bone}${condition.operator}${condition.value}`;
-
-	if ('module' in condition && condition.module != null)
+	} else if ('module' in condition) {
+		Assert(condition.module != null);
 		return `m_${condition.module}${condition.operator}${condition.value}`;
-
-	if ('armType' in condition && condition.armType != null)
+	} else if ('armType' in condition) {
+		Assert(condition.armType != null);
 		return `hand_${condition.armType}_${condition.side}${condition.operator}${condition.value}`;
-
-	AssertNever();
+	} else if ('attribute' in condition) {
+		Assert(condition.attribute != null);
+		return `a_${condition.attribute}`;
+	} else {
+		AssertNever(condition);
+	}
 }
 
 function ParseAtomicCondition(input: string, validBones: string[]): AtomicCondition {
+	if (input.match(/^\s*a_/)) {
+		const attribute = /^\s*a_(!?[-_a-z0-9]+)\s*$/.exec(input);
```

Comment:
Why the space? There can never be a space in here. Also case insensitive (or allowing A-Z) is needed, as literally all our attributes have capitalized names.